### DISCUSSION TITLE
v6 functions.php

### DISCUSSION THREAD

**Discussion by AteS-Github** [bootscore/bootscore#866] [Q&A]
```html
<script src="/servis/assets/vendor/bootstrap/css/bootstrap.min.css">
<script src="/servis/assets/vendor/bootstrap-icons/bootstrap-icons.css">
<script src="/servis/assets/vendor/aos/aos.css">
<script src="/servis/assets/vendor/glightbox/css/glightbox.min.css">
<script src="/servis/assets/vendor/swiper/swiper-bundle.min.css">
<script src="/servis/assets/css/main.css">
```

```html
<script src="/servis/assets/vendor/bootstrap/js/bootstrap.bundle.min.js">
<script src="/servis/assets/vendor/php-email-form/validate.js">
<script src="/servis/assets/vendor/aos/aos.js">
<script src="/servis/assets/vendor/glightbox/js/glightbox.min.js">
<script src="/servis/assets/vendor/swiper/swiper-bundle.min.js">
<script src="/servis/assets/vendor/purecounter/purecounter_vanilla.js">
<script src="/servis/assets/vendor/imagesloaded/imagesloaded.pkgd.min.js">
<script src="/servis/assets/vendor/isotope-layout/isotope.pkgd.min.js">
<script src="/servis/assets/js/main.js">
```

Hello, I couldn't do it. Could you please add it? I can't fully understand this page when I translate it into English.

For functions.php v6, I will put it in bootscore-child, right? / I will put the service folder bootscore-child here, right?

**Comment by crftwrk** [comment]
?

**Reply by AteS-Github** [reply]
Hello, I couldn't do it. Could you please add it? I can't fully understand this page when I translate it into English.

For functions.php v6, I will put it in bootscore-child, right? / I will put the service folder bootscore-child here, right?

**Reply by AteS-Github** [reply]
I can give you ftp if you want

**Reply by crftwrk** [reply]
> For functions.php v6, I will put it in bootscore-child, right? / I will put the service folder bootscore-child here, right?

Correct.

You can send ftp and wp login here https://bootscore.me/private-info-form/ and then post the key in here in this discussion. But again: 

- Why you want to enqueue `<script src="/servis/assets/vendor/bootstrap/js/bootstrap.bundle.min.js">` and `<script src="/servis/assets/vendor/bootstrap/js/bootstrap.bundle.min.js">`? This will create issues because they already enqueued in parent-theme.
- The paths are correct?

I'm asking this to avoid confusion.

**Reply by AteS-Github** [reply]
Those files are in templates, I will not add them, I want to know the correct one

**Reply by crftwrk** [reply]
Please send wp-login and ftp data to the link above. After sending you get a key, post this key here.

**Reply by AteS-Github** [reply]
Adq2LgJ1KGJF7naGgAnS6Dgvyu0RV56O

**Reply by crftwrk** [reply]
Have a look to your child-themes `functions.php`. There you will find now this:

```php
/**
 * Enqueue custom scripts and styles
 */
function child_theme_enqueue_assets() {
  // CSS Files
  //wp_enqueue_style('bootstrap-icons', get_stylesheet_directory_uri() . '/servis/assets/vendor/bootstrap-icons/bootstrap-icons.css');
  wp_enqueue_style('aos', get_stylesheet_directory_uri() . '/servis/assets/vendor/aos/aos.css');
  //wp_enqueue_style('glightbox', get_stylesheet_directory_uri() . '/servis/assets/vendor/glightbox/css/glightbox.min.css');
  //wp_enqueue_style('swiper', get_stylesheet_directory_uri() . '/servis/assets/vendor/swiper/swiper-bundle.min.css');
  //wp_enqueue_style('main-css', get_stylesheet_directory_uri() . '/servis/assets/css/main.css');

  // JavaScript Files
  //wp_enqueue_script('php-email-form-validate', get_stylesheet_directory_uri() . '/servis/assets/vendor/php-email-form/validate.js', array(), null, true);
  wp_enqueue_script('aos-js', get_stylesheet_directory_uri() . '/servis/assets/vendor/aos/aos.js', array(), null, true);
  //wp_enqueue_script('glightbox-js', get_stylesheet_directory_uri() . '/servis/assets/vendor/glightbox/js/glightbox.min.js', array(), null, true);
  //wp_enqueue_script('swiper-js', get_stylesheet_directory_uri() . '/servis/assets/vendor/swiper/swiper-bundle.min.js', array(), null, true);
  //wp_enqueue_script('purecounter', get_stylesheet_directory_uri() . '/servis/assets/vendor/purecounter/purecounter_vanilla.js', array(), null, true);
  //wp_enqueue_script('imagesloaded', get_stylesheet_directory_uri() . '/servis/assets/vendor/imagesloaded/imagesloaded.pkgd.min.js', array(), null, true);
  //wp_enqueue_script('isotope', get_stylesheet_directory_uri() . '/servis/assets/vendor/isotope-layout/isotope.pkgd.min.js', array(), null, true);
  //wp_enqueue_script('main-js', get_stylesheet_directory_uri() . '/servis/assets/js/main.js', array(), null, true);
}
add_action('wp_enqueue_scripts', 'child_theme_enqueue_assets');
```

The only files which are enqueued right now are `aos.js` and `aos.css`, because all other files do not exist in your folders. Folders are empty. Add the other files to the folders step-by-step and remove then step-by-step the `//` comments at the beginning of each line.

**Reply by AteS-Github** [reply]
files appear at this address

/public_html/wp-content/themes/bootscore-child/servis/

I opened it as you said but it still didn't work

**Reply by AteS-Github** [reply]
demo and dowloand  : https://bootstrapmade.com/demo/Presento/

I downloaded it from here

**Reply by crftwrk** [reply]
AOS is enqueued correctly, add the other files and remove `//` comments in `functions.php` when files are added. All other folders are empty right now.

<img width="194" alt="Bildschirmfoto 2024-10-30 um 20 08 07" src="https://github.com/user-attachments/assets/0ff6b50c-089e-4753-9eac-1e7846dea3e9">

**Reply by AteS-Github** [reply]
I did what functions.php said, I checked the css and js, it is there

the images were not loaded, I loaded them but it still gives an error

Uncaught TypeError: Cannot read properties of null (reading 'classList')
    at toggleScrolled (main.js:18:23)

**Reply by crftwrk** [reply]
Now files are there and enqueued correctly. The TypeError is something inside the `main.js`file which has nothing to do with enqueuing files. About the images, check the path in your content.

**Reply by AteS-Github** [reply]
the files are there but they don't work properly
i changed the image addresses on the home page
but it still doesn't look like it should in the demo

**Reply by crftwrk** [reply]
Yes, you have to edit these files to fit to your project, but this might be a tough job. Simply copy & paste them to your theme won‘t work that way.

**Reply by AteS-Github** [reply]
thank you for your efforts and time, I learned a lot from you, I won't be able to use this template, then I'll try another one

good thing you exist :)

**Reply by crftwrk** [reply]
You‘re welcome ;-)
